packages/amplication-data-service-generator/src/register-plugin.ts

```diff
@@ -41,9 +51,10 @@ async function* getPluginFuncGenerator(
  */
 const getAllPlugins = async (pluginList: Plugin[]): Promise<Events[]> => {
   if (!pluginList.length) return [];
+  const allPlugins = [...pluginList, ...defaultPlugins];
   const pluginFuncsArr: Events[] = [];
 
-  for await (const pluginFunc of getPluginFuncGenerator(pluginList)) {
+  for await (const pluginFunc of getPluginFuncGenerator(allPlugins)) {
```

Comment:
I prefer that this logic will not be part of the plugin engine, rather it is better to do it on the entry point of the DSG when preparing all other data for the generation process.  

maybe here: https://github.com/amplication/amplication/blob/a598f4d2861388ae5d43a3058ddd0f162da58023/packages/amplication-data-service-generator/src/create-data-service-impl.ts#L32